src/main/kotlin/se/uulm/snowballr/backend/repository/association/ProjectMemberTableRepo.kt

```diff
@@ -28,7 +28,7 @@ import java.util.UUID
  */
 interface IProjectMemberTableRepo {
     /**
-     * Returns a project member by its composed ID or throws a [NotFoundException] if the project with the passed [projectId] and [userId] doesn't exist.
+     * Returns a project member by its composed id or throws a [NotFoundException] if the project with the passed [projectId] and [userId] doesn't exist.
```

Comment:
This was probably undone accidentally.
```suggestion
     * Returns a project member by its composed ID or throws a [NotFoundException] if the project with the passed [projectId] and [userId] doesn't exist.
```

Reply:
Ah shit. This must have been undone during the rebase mess. Sorry.